docs/guide/configuration/device-availability.md

```diff
@@ -14,15 +14,17 @@ availability:
     enabled: true
 ```
 
-The availability feature works differently for active and passive devices.
+The availability feature works differently for active and passive devices, since passive devices cannot be pinged.
 
-- Active devices (routers or mains powered end devices): by default they have to check-in every 10 minutes. If they
-  don't, they will be pinged, if that fails the device will be marked as `offline`.
-- Passive devices (everything that is not an active device, mostly battery powered devices): these devices need to
-  check-in every 25 hours, they cannot be pinged so if they don't they will be marked as `offline`.
+- Active devices (_non battery-powered_): by default they have to check-in\*\* every 10 minutes.
+  If they don't, they will be pinged, if that fails the device will be marked as `offline`.
+- Passive devices (_battery-powered_): by default they have to check-in\*\* every 25 hours.
+  If they don't they will be marked as `offline`.
 
 Note that this timeout is persisted between Zigbee2MQTT restarts. So if you for example stop Zigbee2MQTT for longer than 10
-minutes, all your active devices will be marked as `offline` initially.
+minutes, all your active devices will be marked as `offline` initially until they check-in\*\* again.
+
+\*\* A check-in is any kind of device-initiated update that reaches Zigbee2MQTT (even internal updates that are not displayed/reported).
```

Comment:
```suggestion
\*\* A check-in is any kind of Zigbee message of the device that reaches Zigbee2MQTT (even internal updates that are not displayed/reported).
```

This might be easier to understand for users